### DISCUSSION TITLE
Song order adjustment

### DISCUSSION THREAD

**Discussion by IVII1** [ppy/osu#25297] [Ideas]
Ever since i started playing osu! the one thing that kinda bothered me was the fact that the "recently played" order bumped the song i just played at the top right after i finished playing it. I have a lot of maps so it's a pain to always have to scroll to find maps i haven't played in a while. Would it be possible/make sense to delay the "bump up" of songs until the end of the session so i can easily navigate maps i haven't played without having to scroll every time? I know it's a bit counter intuitive, just wondering what everyone else's opinion is on this matter.

**Comment by 3r1** [comment]
As someone who frequently uses "Last Played" and "Date Added", a solution I like is collapsible folders for date ranges.

Something similar to:
"Today"
"Yesterday"
"Last 7 Days"
"Last month" (30 days)
"This year"
"Older than this year" or a folder for each year

**So the solution would be: you collapse "Today" and play through what you played at any other time.**

Folders for every month of this year could be nice but I have no strong opinion.

These folders would be useful for navigating in song select through all sorting by date options: Date Added, Date Submitted, Date Ranked, Last Played, and any future date sorting that may be added.
